package.json

```diff
@@ -0,0 +1,5 @@
+{
```

Comment:
this file doesn't look like a properly configured paclasge jason with yarn workspaces.
you have already inclooded uuid dependency into server side.
for making your project a monorepo this package.json must be updated.